### PR TITLE
[author] [new] Rome

### PR DESCRIPTION
**Pull request by bevacqua**
People asked me repeatedly to add this here, so I caved.

https://github.com/bevacqua/rome/issues/51
https://github.com/bevacqua/rome/issues/56

In my opinion there should be some sort of CLI or even web interface allowing you to add libraries here. As it stands, it took me like five minutes because the git repository is so huge (cloning mostly, but even `git status` takes like 10s).

### CODE REVIEW

File: ajax/libs/rome/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "rome",
+  "description": "Customizable date (and time) picker. Opt-in UI, no jQuery!",
+  "homepage": "https://github.com/bevacqua/rome",
+  "version": "1.2.1",
+  "filename": "dist/rome.standalone.min.js",
```
Comment (PeterDaveHello) [inline ajax/libs/rome/package.json]: Should remove `dist` here

File: ajax/libs/rome/package.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      "url": "https://github.com/bevacqua/rome"
+    }
+  ],
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/bevacqua/rome.git",
+    "basePath": "dist",
+    "files": [
+      "rome.css",
```
Comment (PeterDaveHello) [inline ajax/libs/rome/package.json]: Can be only oneline "*" here.

**Comment by bevacqua** [comment]
Made those changes, squashed into one commit.

**Comment by PeterDaveHello** [comment]
Thanks @bevacqua

**Pull request merged**